pkg/image-manager/entities.go

```diff
@@ -68,15 +45,13 @@ type SystemDependencies struct {
 
 // RuntimeDependencies defines a runtime dependency type
 type RuntimeDependencies struct {
-	Format   string `json:"format"`
 	Manifest string `json:"manifest"`
```

Comment:
Why remove format?

Reply:
It's not needed anymore

Reply:
The image building Dockerfile deals with the manifest natively